### DISCUSSION TITLE
Functions action Button only works when you set to global using custom model

### DISCUSSION THREAD

**Discussion by traderpedroso** [open-webui/open-webui#4567] [General]
Action icons only appear on the custom model when I set them in global mode or on the base model."

"For example, when creating a new model, selecting the base model, enabling the selected action on the custom model, but the icon doesn't appear.

**Comment by pipars** [comment]
I see the same.

**Reply by jamie-dit** [reply]
Same here, Required to use https://openwebui.com/f/pad4651/add_to_memories_action_button
